### ISSUE TITLE
[undefined] "Support" button have not blue highlight when user focus on it using tab

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3067]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Press tab key till the "Support" button is selected

**Expected Result**
"Support" button should have blue highlight when user focus on it using tab

**Actual Result**
"Support" button have not blue highlight when user focus on it using tab

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=d3d58f73%2D24c7%2D4199%2D8955%2D30f41c32b22a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fdoremihong%2FBug%5F12
